website/docs/table_types.md

```diff
@@ -36,17 +38,28 @@ Hudi supports the following query types
 
 - **Snapshot Queries** : Queries see the latest snapshot of the table as of a given commit or compaction action. In case of merge on read table, it exposes near-real time data(few mins) by merging
   the base and delta files of the latest file slice on-the-fly. For copy on write table,  it provides a drop-in replacement for existing parquet tables, while providing upsert/delete and other write side features.
-- **Incremental Queries** : Queries only see new data written to the table, since a given commit/compaction. This effectively provides change streams to enable incremental data pipelines.
-- **Read Optimized Queries** : Queries see the latest snapshot of table as of a given commit/compaction action. Exposes only the base/columnar files in latest file slices and guarantees the
+- **Incremental Queries** : Queries only see new data written to the table, since a given commit/compaction. 
+  This effectively provides change streams to enable incremental data pipelines. By default, this produces the latest 
+  snapshot of the changes since a given point in timeline.
+  - ***Incremental Queries(CDC)*** : These are a subtype fo Incremental queries, where queries see all changed data since 
+    a given commit/compaction as opposed to latest state of changed data. This enables full cdc style query use cases 
+    allowing to see before and after images of the changes along with operations that caused the change.
+- **Read Optimized Queries** : Queries see the latest snapshot of table as of a given commit/compaction action. Exposes only the base/columnar files in the latest file slices and guarantees the
   same columnar query performance compared to a non-hudi columnar table.
+- **Bootstrap Queries** : Queries see the latest snapshot of a bootstrapped Hudi table as of a given commit/compaction action.
```

Comment:
@jonvex  can you review this bootstrap querydescription for correctness?

Reply:
Discussed with @jonvex . This can be moved to bootstrapping related page. Resolving the comment here.